gtsfm/evaluation/compare_metrics.py

```diff
@@ -0,0 +1,66 @@
+"""Script to log metrics for comparison across SfM pipelines (GTSfM, COLMAP, etc.) as a JSON.
```

Comment:
this only contains a function? so we don't actually refer to it as script.. although you could call it that. 
Not sure what it does as well.

Reply:
Changed script to function and added more description.